src/tests/e2e/utils/requestInterceptor.js

```diff
@@ -2,11 +2,10 @@
 declare var Base64: any
 declare var log: any
 
-import { erc20TokensAddress } from '../../../erc20Tokens.js'
 const _ = require('lodash')
 const chalk = require('chalk')
 
-const CONTRACT_ADDR = erc20TokensAddress['dai'].toLowerCase()
+const CONTRACT_ADDR = '0x4aacB7f0bA0A5CfF9A8a5e8C0F24626Ee9FDA4a6'.toLowerCase()
```

Comment:
move dai address to a config file and import

Reply:
fixed